### PR TITLE
Changes branch

### PR DESCRIPTION
**Pull request by mumo-esther**
Created a branch and made changes requested by the last reviewer in https://github.com/mumo-esther/Desktop-portfolio/pull/1.

it was about styling the background images around the contact form.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,756 @@
...
+    display: none;
+  }
+
+  .header-title {
+    padding-top: 3em;
+  }
+
+  .header-nav-desktop {
+    display: flex;
+    flex-direction: row;
```
Comment (christianbiring1) [inline style.css:214]: - [ ] Kindly make sure your mobile version remains intact. As you can on your page how it looks on small size. we do not need to scroll to the right to see the content of the page.
Your design :point_down: 
image
Figma design :point_down: 
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,756 @@
...
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    padding-top: 1em;
+    margin: 0;
```
Comment (christianbiring1) [inline style.css:219]: - [ ] Kindly stick to the Figma template as much as you can.
YOur design :point_down: 
image

Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @mumo-esther , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
almost


 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary`
- [ ] Good commit messages.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -743,13 +745,19 @@ footer::after {
 }
 
 @media screen and (max-width: 767px) {
-  .card-categories {
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ Optional ] As stated in the project Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider). Kindly adjust it.
- [ Optional ] The Minimum mobile screen size should be 375px.So any Errors in screen sizes less than 375px should be ignored.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @mumo-esther       
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Good README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**